test/services/FinancialDataServiceSpec.scala

```diff
@@ -16,21 +16,20 @@
 
 package services
 
-import connectors.DirectDebitConnector
+import connectors.FinancialDataConnector
 import models.DirectDebit
 import models.errors.UnexpectedError
 import org.mockito.Mockito.when
 import org.scalatestplus.mockito.MockitoSugar
 import play.api.http.Status.INTERNAL_SERVER_ERROR
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.test.UnitSpec
-
 import scala.concurrent.Future
 
-class DirectDebitServiceSpec extends UnitSpec with MockitoSugar {
+class FinancialDataServiceSpec extends UnitSpec with MockitoSugar {
```

Comment:
Can you add unit tests in here for the new function you have added. These ones will currently be very simple as you can see from the DD tests below.